community/samples/serving/helloworld-r/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7
```

Comment:
1. Why not use something like https://shiny.rstudio.com/? Fair enough if this is a more typical pattern.

2. Kind of need to see Python and R together here, but reasons why helloworld-shell uses go:

  * Small resulting image, as the go runtime does not need to be included once the binary is produced.
  * Higher performance.

I think it's a fair counter-argument that R developers will be more likely familiar with Python.

Reply:
1. Shiny/rstudio from my research, runs the studio on a server - no actual script running or server request/response capabilities. 
2. I went back and forth on this. R users are going to have no experience with Go, but they will know what Python is and looks like. Though we can make the argument that they may never have to touch the Go code for this to work.